BookingsApi/BookingsApi.Domain/Hearing.cs

```diff
@@ -79,7 +79,11 @@ protected Hearing()
 
         public abstract HearingMediumType HearingMediumType { get; protected set; }
         public virtual HearingVenue HearingVenue { get; protected set; }
+        
+        [Obsolete("Use HearingVenue.Name instead")]
         public virtual string HearingVenueName { get; set; }
```

Comment:
is it safe to delete this property now and make sure the ef mappings keep the column, if still needed for the migration?

Reply:
I don't think we can unfortunately, the property still needs to be there in order to be referenced by the mapper. We've had issues in the past removing mapped properties due to how the snapshot is generated